salt/journal/init.sls

```diff
@@ -117,25 +109,39 @@ composer-install:
         {% endif %}
         - cwd: /srv/journal/
         - user: {{ pillar.elife.deploy_user.username }}
-        # to correctly write into var/
-        - umask: 002
         - env:
-            - SYMFONY_ENV: {{ pillar.elife.env }}
             - COMPOSER_DISCARD_CHANGES: 'true'
         - require:
             - file: config-file
-            - journal-cache-clean
             - journal-php-extensions
 
+journal-cache-clean:
+    cmd.run:
+        - name: rm -rf var/cache/
+        - cwd: /srv/journal/
+        - require:
+            - var-directory
+
+journal-cache-warmup:
+    cmd.run:
+        - name: bin/console cache:warmup
+        - cwd: /srv/journal/
+        - user: {{ pillar.elife.webserver.username }}
+        - env:
+            - APP_ENV: {{ pillar.elife.env }}
+        - require:
+            - composer-install
+            - journal-cache-clean
+
 journal-assets-install:
     cmd.run:
         - name: bin/console assets:install --symlink
         - cwd: /srv/journal/
         - user: {{ pillar.elife.deploy_user.username }}
         - env:
-            - SYMFONY_ENV: {{ pillar.elife.env }}
+            - APP_ENV: {{ pillar.elife.env }}
```

Comment:
Change in https://github.com/elifesciences/journal/pull/940, but would have had the same result (just always as the `dev` environment).